### DISCUSSION TITLE
Quick Tag UI being below reblog button -SOLVED/EXPLAINED

### DISCUSSION THREAD

**Discussion by quietlyrebelling** [AprilSylph/XKit-Rewritten#631] [Suggestion Box]
So this is a very minute change but something I definitely notice constantly. I originally used xKit for Chrome but just recently switched to Firefox. The Quick Reblog UI for Chrome always popped up ABOVE the reblog button/where your mouse is hovering whereas on Firefox it always pops up below. I find this to be a small but noticeable inconvenience and wondered if there were plans to change it. In my opinion it was better to have the Quick Reblog box pop up above that way it wasnt cut off the bottom of the page.

**Comment by AprilSylph** [comment]
The Quick Reblog position is a preference you can set. I recommend importing your XKit configuration from Chrome via the Backup tab in the XKit Rewritten control panel.

**Reply by quietlyrebelling** [reply]
I feel so silly lol this is what I get for doing this at 3am last night. Thank you so much!
